src/parsers/i3d/unpackGeometry/main.ts

```diff
@@ -35,61 +34,46 @@ export function unpackPrimitives(
         primitiveCompressedData,
         uncompressedValues,
         maps,
-        primitivesPerSectorAndChildren,
+        primitivesPerSector,
         filterOptions
       );
     });
   }
+  for (const primitiveGroup of rootSector.traversePrimitiveGroups()) {
+    primitiveGroup.sort();
+  }
 }
 
-function countRenderedPrimitivesPerSectorAndChildren(
-  sector: Sector,
-  compressedData: PerSectorCompressedData,
-  primitivesPerSectorAndChildren: PrimitivesPerSectorAndChildren
-) {
-  // Count rendered primitives in sector
-  primitivesPerSectorAndChildren[sector.path] = {};
-  RenderedPrimitiveNames.forEach(renderedPrimitive => {
-    primitivesPerSectorAndChildren[sector.path][renderedPrimitive] = 0;
-  });
-
-  compressedData[sector.path].primitives.forEach(fileGeometryData => {
-    renderedPrimitivesPerFilePrimitive[fileGeometryData.type].forEach(renderedPrimitiveInfo => {
-      primitivesPerSectorAndChildren[sector.path][renderedPrimitiveInfo.name] +=
-        renderedPrimitiveInfo.count * fileGeometryData.count;
+function countRenderedPrimitivesPerSector(rootSector: Sector, compressedData: PerSectorCompressedData) {
+  const primitivesPerSector: PrimitivesPerSector = {};
```

Comment:
If this is a count, use `count` or `num`, or `numberOf` in the variable name maybe?